src/commands/createStaticWebApp/tryGetApiLocations.ts

```diff
@@ -16,16 +16,17 @@ const hostFileName: string = 'host.json';
  * If a single function project is found, returns that path.
  * If multiple projects are found, will prompt
  * @param workspaceFolder Per the VS Code docs for `findFiles`: It is recommended to pass in a workspace folder if the pattern should match inside the workspace.
+ * @param shallow Search at the root, and on level down only.
  */
-export async function tryGetApiLocations(context: IActionContext, workspaceFolder: WorkspaceFolder | string): Promise<string[] | undefined> {
-    return await telemetryUtils.runWithDurationTelemetry(context, 'tryGetProject', async () => {
+export async function tryGetApiLocations(context: IActionContext, workspaceFolder: WorkspaceFolder | string, shallow?: boolean): Promise<string[] | undefined> {
+    return await telemetryUtils.runWithDurationTelemetry(context, `tryGetProject${shallow ? 'Shallow' : ''}`, async () => {
```

Comment:
One nit thing:  It might be easier to just make "shallow" a telemetry property.  I think it's easier to grab all the events and check the property than filtering out by eventName